.github/workflows/release_pr.yml

```diff
@@ -0,0 +1,58 @@
+name: Create PR for a release
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'Release from which branch?'
+        required: true
+        default: 'main'
+      git_user_name:
+        description: 'Git user name'
+        required: true
+        default: 'devops'
+      git_user_email:
+        description: 'Git user name'
+        required: true
+        default: 'devops@test.com'
+jobs:
+  create_pr_for_next_release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Update git
+      run: |
+        sudo add-apt-repository -y ppa:git-core/ppa
+        sudo apt-get update
+        sudo apt-get install git -y
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.inputs.base_branch }}
+        token: ${{ github.token }} # This will need to change to a token belonging to a service account.
+        fetch-depth: 0
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7
+    - name: Install dependencies
+      run: |
+        cd scripts
+        gem install bundler
+        bundle install
+    - name: Configure git options
+      run: |
+        cd scripts
+        bundle exec fastlane android configure_git_options git_user_email:${{ github.event.inputs.git_user_email }} git_user_name:${{ github.event.inputs.git_user_name }}
+    - name: Create/checkout a branch for the release
+      run: |
+        branch_name=releases/`date "+%Y%m%d"` 
```

Comment:
Two issues I see with this naming strategy:
 - If we try to do two releases on the same day, this may fail because it will try to create a branch that already exists.
 - We may end up with a bunch of stale branches on the repo, since this creates a new one for each release.

Could we instead just keep it simple, and always use the same branch name (e.g `release`), and just always do a `git branch -D release` before trying to create it?

Reply:
I like that...I'll make the change.

Reply:
Branching strategy updated. Workflow will try to delete a branch called `release` before creating a new `release` branch from the latest `main`